File: SEM4-C/ejer3.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SEM4_C
{
    internal class ejer3
    {
        static void Main(string[] args)
        {
            Console.WriteLine("--- Sistema de conversión monetario ---\n");

            Console.Write("Ingrese el monto en soles: ");
            double mo = double.Parse(Console.ReadLine());

            Console.WriteLine("\n1. Convertir a dolares");
            Console.WriteLine("2. Convertir a euros\n");

            Console.Write("Ingrese una opción: ");
            int opc = int.Parse(Console.ReadLine());

            switch (opc)
            {
                case 1:
                    double dolares = mo/3.75;
                    Console.WriteLine("\nDolares: "+dolares);
                    break;
                case 2:
                    double euros = mo / 4.05;
                    Console.WriteLine("\nEuros: " + euros);
                    break;
                default:
                    Console.WriteLine("Opción incorrecta!");
                    break;
            }
        }
    }
}

File: SE4-P/ejer3.py
print("--- Sistema de conversión monetario ---\n")

mo = float(input("Ingrese el monto en soles: "))

print("\n1. Convertir a dolares")
print("2. Convertir a euros\n")

opc = int(input("Ingrese una opción: "))

match opc:
    case 1:
        dolares = mo/3.75
        print("\nDolares: ",round(dolares,2))
    case 2:
        euros = mo / 4.05
        print("\nEuros: ",round(euros,2))
    case _: print("\nOpción incorrecta!")


